doc/persistent_scenes_and_states/persistent_scenes_and_states.rst

```diff
@@ -0,0 +1,59 @@
+Persistent Planning Scenes and Robot States
+===========================================
+
+The "MotionPlanning" plugin of RViz offers the possibility to save
+complete planning scenes and robot states persistently.
+Currently, two storage plugins (based on
+`warehouse_ros <https://github.com/ros-planning/warehouse_ros>`_) are available:
+
+* `warehouse_ros_mongo <https://github.com/ros-planning/warehouse_ros_mongo>`_, which uses MongoDB as backend (default)
+* `warehouse_ros_sqlite <https://github.com/gleichdick/warehouse_ros_sqlite>`_, which uses SQLite as backend
+
+You can install both of them with your favourite package manager
+(e.g. ``apt-get install ros-foxy-warehouse-ros-mongo``) or
+`build them from source <../getting_started/getting_started.html>`_
+(of course, you'll have to check out the corresponding repository into your ``src`` folder for that).
+
+Storage plugin selection
+------------------------
+
+The storage plugin is determined by the parameter ``warehouse_plugin``.
+Valid options are ``warehouse_ros_mongo::MongoDatabaseConnection`` for MongoDB and
+``warehouse_ros_sqlite::DatabaseConnection`` for SQLite.
+Furthermore, the parameters ``warehouse_host`` and ``warehouse_port`` configure the connection details.
+In case of the SQLite plugin, ``warehouse_host`` contains the path of the database file.
+They have to be set in the launch file (e. g. ``move_group.launch.py``) as followed:
+
+.. tutorial-formatter:: ./move_group.launch.py
+
+You can find the launch file :codedir:`here in the tutorials GitHub repository <persistent_scenes_and_states/move_group.launch.py>`
+or (unmodified) in your workspace at ``install/moveit2_tutorials/share/moveit2_tutorials/launch/demo.launch.py``.
+
+Connecting to the storage backend
+---------------------------------
+
+After choosing the storage plugin and configuring the launch.py file,
+run RViz using ::
+
+   ros2 launch moveit2_tutorials demo.launch.py
+
+In RViz, navigate to the "Context" tab of the "MotionPlanning" window.
+Verify the connetion details (host/port for MongoDB, file path for SQLite)
+and click on "Connect".
+
+.. image:: rviz_connect.png
+    :width: 600px
+
+After that, a dialogue box will appear and ask you whether you'd like to erase all current
+states and scenes in RViz (not in the database, the persistent data is not affected by that).
```

Comment:
Conceptually It shouldn't do that because the list is/lists are still empty. Briefly looking at the code, without testing a plugin, I'm not sure which MessageBox is triggered here but it should be relatively easy to add the additional check.

Alternatively, please add
```suggestion
states and scenes in RViz (not in the database, the persistent data is not affected by that).
As you just started RViz, you can safely select "yes".
```